test/integration/targets/azure_rm_storageaccountkeys_facts/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+ - name: Create storage account name
```

Comment:
Thanks for the tests, it may be better to just lump them together with the `azure_rm_storageaccountkeys` target as that "should" already be creating the storage accounts for us to test the module against. I haven't looked at those tests so not sure if that is viable or not.

Reply:
I don't see any azure_rm_storageaccountkeys class. Do you mean azure_rm_storageaccount?

Reply:
actually, this module is a little bit different than other facts modules.
Keys can't be updated by any API, they can be only changed by using "regenerate" call of REST API (not sure what's the Python equivalent). So basically this is one way operation. Perhaps azure_rm_storageaccount should provide some way of regenerating keys. I am not sure how that should work.
So here "raw" / "curated" doesn't really make much sense.
"Curated" facts will be a dictionary, while "raw" will be an array looking similar to this:

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/Microsoft.Storage/stable/2018-02-01/examples/StorageAccountListKeys.json

@jborean93 do you think we should still have raw/curated here? perhaps for consistency.

Reply:
My apologies I should have looked first, we should definitely be doing the curated format first over raw so when the module is first created we can default to the curated format.